src/api/app/controllers/webui/packages/meta_controller.rb

```diff
@@ -1,9 +1,12 @@
 module Webui
   module Packages
     class MetaController < Webui::WebuiController
+      include ScmsyncChecker
+
       before_action :set_project
       before_action :set_package
 
+      before_action :check_scmsync
```

Comment:
```suggestion
      before_action :check_scmsync
      before_action :set_package

```

I would move the check for scmsync on a project just after setting the project and just before setting the package...